sources/packages/web/src/components/common/applicationTracker/Assessment.vue

```diff
@@ -23,6 +30,7 @@ import { AssessmentTriggerType } from "@/types";
 import { defineComponent, PropType } from "vue";
 
 export default defineComponent({
+  emits: ["goToNoticeOfAssessment"],
```

Comment:
Minor, we have been opting by using the `emits` using the object syntax instead of the string.